.travis.yml

```diff
@@ -9,4 +9,25 @@ scala:
   - 2.12.8
 
 script:
+  - npm install
+  - npm run build
   - sbt dist
+
+#NOTE: Currently all branches will deploy to the development AWS environment for the purposes of Alpha
+deploy:
+  - provider: script
+    skip_cleanup: true
+    script: bash deploy.sh dev
+    on:
+      all_branches: true
```

Comment:
Do we definitely want it to deploy every branch to the dev environment? I just wonder if it'll be confusing because you might start testing something, and then not realise that your deployment's been overwritten by someone else's branch because they pushed a minute later.